contracts/cairo/src/Escrow.cairo

```diff
@@ -28,6 +28,10 @@ trait IEscrow<ContractState> {
     fn set_herodotus_facts_registry_contract(
         ref self: ContractState, new_contract: ContractAddress
     );
+    fn get_mm_starknet_contractv2(self: @ContractState) -> ContractAddress;
+    fn set_herodotus_facts_registry_contractv2(
```

Comment:
shouldn't 'v2' functions be in Escrowv2.cairo ?

Reply:
My bad, thanks uri